hg-fast-export.py

```diff
@@ -128,11 +128,13 @@ def get_author(logmessage,committer,authors):
   return committer
 
 def remove_gitmodules(ctx):
-  """Removes all submodules"""
-  # Remove all submodules as we don't detect deleted submodules properly
-  # in any other way. We will add the ones not deleted back again below.
-  for module in submodule_mappings.keys():
-    wr('D %s' % module)
+  """Removes all submodules of ctx parents"""
+  # Not quite sure whether submodules from all parents should be removed.
+  # Subrepos from first parent may be sufficient.
```

Comment:
How does the earlier version's parsed `.hgsubstate` file differ from using all the parents? Is it equivalent? Have you analyzed this?

Reply:
From your reasoning it is clear that removing the submodules from all parents should be safe, make the comment more assertive so the next contributor won't have analyze the code again. I'm thinking: "Removing all submoduies coming from all parents is safe, as the submodules of the current commit will be re-added below. A possible optimization would be to only remove the submodules of the first parent"

Reply:
Please line break the comment at 80 columns. The comment, as written in c1089b015442ae2ae3834bcc3a2147036a7b93ff, is not grammatically correct, why not simply use what I suggested above?